### ISSUE TITLE
GIF label is now dark-on-grey on dark theme

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#25836]
### Steps to reproduce

Compound seems to have made dark-theme EW show the GIF label on images as black-on-grey:

<img width="82" alt="Screenshot 2023-07-22 at 17 34 45" src="https://github.com/vector-im/element-web/assets/1294269/15a22332-c2cd-43d6-920b-cdcdb4340e14">

#### What did you expect?

legible label

### Application version

July 22 nightly

### Will you send logs?

No

**Comment by germain-gg** [comment]
@janogarcia here's another example of where we need the inverted tokens
